doc/man/8/ceph.rst

```diff
@@ -248,6 +248,15 @@ Usage::
 	ceph df {detail}
 
 
+features
+--------
+
+Show the releases and supported features of all connected daemons and clients, along with the numbers of them in each bucket grouped by the corresponding features/releases.
```

Comment:
We should add a bit of background here.  Each release of ceph supports a different set of features, expressed by the features bitmask.  New cluster features require that clients support the feature or else they are not allowed to connect to the features.  As new features or capabilities are enabled after an upgrade, older clients are prevented from connecting.  

The features and releases of all clients connected to the cluster can be seen with::

  ceph features

Note that this only described clients that are currently connected.  Operators should take care to ensure there aren't other client hosts that are currently offline; they may also need to be upgraded.